plugins/lookup/hashi_vault.py

```diff
@@ -186,18 +194,22 @@
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello:value auth_method=userpass username=myuser password=psw url=http://myvault:8200') }}"
 
-- name: Using an ssl vault
+- name: Connect to Vault using TLS
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hola:value token=c975b780-d1be-8016-866b-01d0f9b688a5 validate_certs=False') }}"
 
 - name: using certificate auth
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret/hi:value token=xxxx url=https://myvault:8200 validate_certs=True cacert=/cacert/path/ca.pem') }}"
 
-- name: authenticate with a Vault app role
+- name: Authenticate with a Vault app role
   ansible.builtin.debug:
     msg: "{{ lookup('community.general.hashi_vault', 'secret=secret/hello:value auth_method=approle role_id=myroleid secret_id=mysecretid') }}"
 
+- name: Authenticate with a JWT
+  debug:
+      msg: "{{ lookup('hashi_vault', 'secret=secret/hello:value auth_method=jwt role_id=myroleid jwt=myjwt url=https://myvault:8200')}}"
+
```

Comment:
Many of the examples that require newer versions of this plugin are toward the bottom. I'm not opposed to doing away with that convention, but if we want to do that then the comment should be removed from those as well.

I'd prefer newer examples shy away from the term string arguments in favor of using real arguments to the plugin (not embedding them in the term string), see the later examples.

Reply:
Moved and adjusted the example as suggested. (And fixed a minor bug in the 'aws_iam_login' example.)